lib/membrane_portaudio_plugin/source.ex

```diff
@@ -34,6 +34,11 @@ defmodule Membrane.PortAudio.Source do
                 spec: :low | :high,
                 default: :high,
                 description: "Latency of the output device"
+              ],
+              max_channels: [
+                spec: 1..2,
+                default: 2,
+                description: " Max number of channels that the device will be allowed to output "
               ]
```

Comment:
I'd prefer a `channels` option instead, that would:
- provide a given number of channels if set
- raise if the device doesn't support that
- if not set, provide stereo if available and mono otherwise